### PR TITLE
[15.0][MIG] project_category

### PR DESCRIPTION
**Pull request by emagdalenaC2i**

### CODE REVIEW

**Review by AaronHForgeFlow — COMMENTED** [review]
`2022-01-26 07:03:13,040 159 WARNING be0bfbc25-a776-494d-bb83-c6dd305daa23 py.warnings: /opt/odoo/odoo/fields.py:722: UserWarning: Field project.type.complete_name should be declared with recursive=True`

**Review by Mantux11 — APPROVED** [review]

File: project_category/models/project_type.py
Comment (Mantux11) [inline project_category/models/project_type.py]:
```suggestion
    complete_name = fields.Char(
        compute="_compute_complete_name", 
        recursive=True,
        store=True
    )
```

Sorry. I missed the error. @AaronHForgeFlow  showed the error correctly. There is suggestion :slightly_smiling_face:

Comment (emagdalenaC2i) [reply project_category/models/project_type.py]: @AaronHForgeFlow and @Mantux11 Done

**Comment by emagdalenaC2i** [comment]
@AaronHForgeFlow Please, could you update your review?

**Review by Mantux11 — APPROVED** [review]
Now, it LGTM.

**Review by AaronHForgeFlow — APPROVED** [review]
Code review LGTM

I am waiting for the merge for the actual users of the module to review as long as I am not a user of the module.

**Comment by emagdalenaC2i** [comment]
@OCA/project-service-maintainers  Please, could you merge this PR?

**Comment by rafaelbn** [comment]
@emagdalenaC2i , hello! 😄 .

Are you aware about all the improvement in projects like: (in project not in tasks)

- tags (project.tags)
- milestones (project.milestones)
- stages (project.stages)
- updates (project.update)

I ask because in this moment I don't remember why this module wasn't named as project_type like the field it adds: https://github.com/OCA/project/pull/872/files#diff-fc3e3e1cbbc4ab655bdf4cceaa4a631c4a67a67a34ee49345dfcf20f1613acb1R10

Now with this several improvements the name could confuse.

Just a question!

Take a look in https://github.com/odoo/odoo/blob/15.0/addons/project/models/project.py

**Comment by emagdalenaC2i** [comment]
@rafaelbn I agree. I think it should be rename to project_type
I just did the migration, so I would like to know what other users think

**Comment by rafaelbn** [comment]
Dear @dreispt @Saran440 @Nikul-Chaudhary @tarteo @cubells @pedrobaeza what do you think?

It is just a question if you want to maintain we can merge as is 😄 

Sorry 🙏 I closed by mistake

**Review by Mantux11 — APPROVED** [review]

**Comment by AaronHForgeFlow** [comment]
After the last change I think the module should change the name, put that in the readme and do the renaming in OpenUpgrade

**Comment by Saran440** [comment]
I agree, module name should be changed to `project_type`, as it easier than before.

**Pull request closed without merging**
